shared/src/test/scala/squants/market/MoneySpec.scala

```diff
@@ -152,17 +152,39 @@ class MoneySpec extends FlatSpec with Matchers {
     y.moneyMin(x) should be(y)
   }
 
-  it should "return proper result when adding like currencies with no MoneyContext in scope" in {
+  it should "return a proper result when adding like currencies with no MoneyContext in scope" in {
     USD(1) + USD(2) should be(USD(3))
     USD(1).plus(USD(2)) should be(USD(3))
   }
 
-  it should "return proper result when subtracting like currencies with no MoneyContext in scope" in {
+  it should "return a proper result when subtracting like currencies with no MoneyContext in scope" in {
     USD(5) - USD(2) should be(USD(3))
     USD(5).minus(USD(2)) should be(USD(3))
   }
 
-  it should "return proper result when dividing like currencies with no MoneyContext in scope" in {
+  it should "return a proper result with no additional precision loss when multiplying by a primitive numeric" in {
```

Comment:
That's a good suggestion in general for test, scala check could be appropriate for that